From 2e392063c74a19101e5885665fbc1f6f986a3002 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Mon, 8 Feb 2021 10:16:30 +0100
Subject: [PATCH] Add possibility to not install systemd service files

Some systems might use elogind instead and thus don't need the service
files
---
 src/daemon/daemon.pro | 8 +++++---
 systemd/systemd.pri   | 9 +++++----
 2 files changed, 10 insertions(+), 7 deletions(-)

diff --git a/src/daemon/daemon.pro b/src/daemon/daemon.pro
index c52089a..dcc3109 100644
--- a/src/daemon/daemon.pro
+++ b/src/daemon/daemon.pro
@@ -39,6 +39,8 @@ systemd.files = \
         systemd/nemo-devicelock.service
 systemd.path= /usr/lib/systemd/system
 
-INSTALLS += \
-        systemd \
-        target
+!no_systemd{
+	INSTALLS += systemd
+}
+
+INSTALLS += target
diff --git a/systemd/systemd.pri b/systemd/systemd.pri
index bb6fd8f..047272f 100644
--- a/systemd/systemd.pri
+++ b/systemd/systemd.pri
@@ -1,4 +1,3 @@
-
 systemd.files = \
         $$PWD/nemo-devicelock.socket
 systemd.path= /usr/lib/systemd/system
@@ -7,6 +6,8 @@ policy.files = \
         $$PWD/org.nemomobile.devicelock.conf
 policy.path = /etc/dbus-1/system.d
 
-INSTALLS += \
-        policy \
-        systemd
+!no_systemd{
+	INSTALLS += systemd
+}
+
+INSTALLS += policy
-- 
2.23.0

